### ISSUE TITLE
[Bug] No `.strings` files in add translation view in my project

### ISSUE THREAD

**Issue by Jeehut** [FlineDev/ReMafoX#5]
I can't add new keys, it says, there are no .string files in my project, but I'd not true, it saw them before..
<img width="898" alt="Screenshot 2022-08-26 at 11 59 59" src="https://user-images.githubusercontent.com/873387/186868130-963851ae-0e20-4157-97f0-79ab01bd7124.png">

Is there some cache or something? Can I delete info about project and restart setup?

_Originally posted by @anivaros in https://github.com/FlineDev/ReMafoX/issues/3#issuecomment-1228247512_

**Comment by Jeehut** [comment]
@anivaros You can simply press "Rescan Strings(dict) files" to rescan your project for all Strings(dict) files. If that doesn't work, you should check your "Search Paths" in the config file. This can happen when files are moved around or paths change (e.g. when the config file is moved or the Xcode project is moved). Please tell me if this works.

**Comment by anivaros** [comment]
I've tried to press button of course 🙂, but nothing happens..

**Comment by anivaros** [comment]
I think I could broke it by renaming project (removed space between words)

**Comment by Jeehut** [comment]
@anivaros If you renamed the project, then please try just removing the app from the "Project Browser" window like this:
![Screen Shot 2022-08-26 at 12 38 26](https://user-images.githubusercontent.com/6942160/186886366-cb30474b-8500-4fa9-870b-61b369e261ed.png)

Then, simply reopen the `.remafox` config file by double-pressing it and then pressing "Quick Setup".

If that doesn't help, you can also try removing the app from the "Project Browser" **and** deleting the `.remafox` config file and setting the project up freshly.

Also, it could help me understand the problem better if you provided me:
1. A screenshot of the "Search Paths" config file page.
2. The path to your `.remafox` config file
3. The path to a `.strings` file you would expect to see in the list

**Comment by anivaros** [comment]
I have no delete button in "Project Browser":
<img width="912" alt="Screenshot 2022-08-26 at 13 51 25" src="https://user-images.githubusercontent.com/873387/186888899-076cab5e-522d-4371-99f7-a92de129f49d.png">

I found problem, I've added slash in the end of Search path. I removed it and it fixed the problem.

Buuuuuut..... now crash on generate enum is back lol 😅

**Comment by Jeehut** [comment]
@anivaros Oh, judging from the looks of your screenshot, I think there are actually some issues with macOS Ventura here. (You said you're on Ventura, [here](https://github.com/FlineDev/ReMafoX/issues/3#issue-1345490277).) Cause the whole border isn't supposed to be there and there's also no delete button, as you stated, but it's supposed to be there.

It looks like I have to test things with Ventura to help you, I didn't test on Ventura yet. But will do that before the next beta.

As for the issue with the crash on generate enum: Can you please create a separate issue for that and provide the info I had asked for in [this comment](https://github.com/FlineDev/ReMafoX/issues/3#issuecomment-1228090648)? Thank you!

**Comment by anivaros** [comment]
I added comment to issue with crash yesterday. You will reopen it if crash reproduces again.

**Comment by Jeehut** [comment]
@anivaros I just tested the app on macOS Ventura and I can reproduce the issue with how the project browser is looking. I'm not sure if I can provide a proper fix before switching to Xcode 14 and I can only switch to Xcode 14 once Apple accepts apps from Xcode 14 for the Mac App Store (which I believe it currently doesn't).

Thus, for the time being, please try deleting the preferences of the app by running this command in Terminal:
```bash
rm ~/Library/Group\ Containers/767S6EFMJ8.dev.fline.Remafox/Library/Preferences/767S6EFMJ8.dev.fline.Remafox.plist
```

This should be similar to deleting your project from the list in the project browser. After doing this, please re-setup your project. This should hopefully fix any issues after renaming the project folder.

Please report if this helps. Thank you for your patience.

**Comment by anivaros** [comment]
>  I can only switch to Xcode 14 once Apple accepts apps from Xcode 14 for the Mac App Store
Why you can't install second Xcode version?

btw, deleting settings helped to remove duplicate project, thanks!

**Comment by Jeehut** [comment]
@anivaros I already have Xcode 14 beta 6 installed. But to fix the UI on Ventura, I have to switch to Xcode 14 as the IDE that I build my Beta apps with as the changes I need to make are not supported by Xcode 13. But if I do that, I can't upload the app to the Mac App Store (which I plan to do very soon) because Apple doesn't accept builds to the App Store from beta Xcode. At least that's how I know it all works, correct me if I'm wrong.

Great to hear that you could remove your duplicate project. After setting the project up from the beginning, do you still have the issue that the "Add translation" view doesn't show your `.strings` files in the selector?

**Comment by anivaros** [comment]
> But to fix the UI on Ventura, I have to switch to Xcode 14 as the IDE that I build my Beta

Oh, I've got it now

> do you still have the issue that the "Add translation" view doesn't show your .strings files in the selector?

No, I've found how to fix this [before](https://github.com/FlineDev/ReMafoX/issues/5#issuecomment-1228358983):
> I found problem, I've added slash in the end of Search path. I removed it and it fixed the problem.

My search path was `CookingRecipes/`, and that's was a problem :)

**Comment by Jeehut** [comment]
@anivaros Oh yes, of course, you already mentioned that, I forgot. I think I kept this open so I could check for the other Ventura issues, and I already did that. I will make sure to fix them as soon as I can entirely switch to Xcode 14, but that's not related to this issue.

So, happy to hear that this was just an issue with your paths setup, closing then. 🎉 

For everyone stumbling upon the same issue: Use the "Preview matching files" button in the config file to check if your specified search paths (and exclude paths) are finding the expected files successfully.
